gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v3_20_24/schema.yml

```diff
@@ -7,3 +7,7 @@ databaseChangeLog:
             tableName: ${gravitee_prefix}flows
```

Comment:
Changing this file might cause some start up issues because of a failing checksum validation during liquibase execution.